### PR TITLE
Graph event model blog post

### PR DESCRIPTION
**Pull request by dilyand**

### CODE REVIEW

**Comment by Mandelliant** [comment]
@dilyand will review this today

File: _posts/2018-03-21-building-a-model-for-atomic-event-data-as-a-graph/index.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+Graphs excel at modelling networks -- social, communication, supply, etc. You can even model and link different types of networks in an overarching graph, which is very powerful. In that environment, events would be best represented by relationships between entities in those networks.
+
+Since graphs are so flexible, the advice is usually to build a model that suits the queries you want to run. Snowplow users come from all sorts of industries and are trying to solve a great variety of challenges. Whatever our model, it needs to be able to accommodate that diversity.
+
+### Event grammar
+
+We've already done great [work][link2] on developing a generalised event grammar that goes some way towards a graph data model for event data. In that model, an event is a snapshot of set of entities in time, eg a subject, an object, context, etc.
```
Comment (bogaert) [inline _posts/2018-03-21-building-a-model-for-atomic-event-data-as-a-graph/index.md]: I agree we have done some great work here, but I'd rephrase this a little to not sound like we're congratulating ourselves :)

**Comment by dilyand** [comment]
Made edits as requested.

@Mandelliant I haven't added any tags or categories. It would be great if you could suggest some.

**Comment by Mandelliant** [comment]
@dilyand sure thing, happy to

File: _posts/2018-03-21-building-a-model-for-event-data-as-a-graph/index.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+In this post:
+
+1. [Why graph databases?](#why-graph-databases)
+2. [How do we model event data as a graph?](#how-do-we-model-event-data-as-a-graph)
+3. [Taking an event-grammar approach](#taking-an-event-grammar-approach)
+4. [Taking a time-series or event-graph approach](#taking-a-time-series-or-event-graph-approach)
+5. [Events as nodes versus events as edges](#events-as-nodes-versus-events-as-edges)
+6. [Do we only want to load atomic event data in a graph database?](#do-we-only-want-to-load-atomic-event-data-in-a graph-database)
+7. [What should be our pilot storage target?](#what-should-be-our-pilot-storage-target)
+
```
Comment (alexanderdean) [inline _posts/2018-03-21-building-a-model-for-event-data-as-a-graph/index.md]: You missed 8., the next post in the series

Comment (Mandelliant) [reply _posts/2018-03-21-building-a-model-for-event-data-as-a-graph/index.md]: @dilyand @alexanderdean I'll add it in- but otherwise it looks good on my end for posting.

Comment (dilyand) [reply _posts/2018-03-21-building-a-model-for-event-data-as-a-graph/index.md]: I left it out on purpose (just felt it didn't belong) but am agnostic about adding it. Thanks @Mandelliant !
